/**:
  ros__parameters:
    resample_num: 20
    time_resolution: 0.5

    target_state:
      lateral_positions: [-4.5, -3.5, -2.5, -1.0, 0.0, 1.0, 2.5, 3.5, 4.0]
      # lateral_positions: [-4.5, -2.5, 0.0, 2.5, 4.0]
      lateral_velocities: [0.0]
      lateral_accelerations: [0.0]
      longitudinal_positions: [0.0]
      longitudinal_velocities: [0.0]
      longitudinal_accelerations: [-0.4, -0.3, -0.2, -0.1, 0.0, 0.1, 0.2, 0.3, 0.4]
      # longitudinal_accelerations: [-0.2, -0.1, 0.0, 0.1, 0.2]

    score_weight: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
    time_decay_weight:
      s0: [1.0, 0.8, 0.64, 0.51, 0.41, 0.33, 0.26, 0.21, 0.17, 0.13]
      s1: [1.0, 0.8, 0.64, 0.51, 0.41, 0.33, 0.26, 0.21, 0.17, 0.13]
      s2: [1.0, 0.8, 0.64, 0.51, 0.41, 0.33, 0.26, 0.21, 0.17, 0.13]
      s3: [1.0, 0.8, 0.64, 0.51, 0.41, 0.33, 0.26, 0.21, 0.17, 0.13]
      s4: [1.0, 0.8, 0.64, 0.51, 0.41, 0.33, 0.26, 0.21, 0.17, 0.13]
      s5: [1.0, 0.8, 0.64, 0.51, 0.41, 0.33, 0.26, 0.21, 0.17, 0.13]

    grid_seach:
      min: 0.1
      max: 1.0
      resolusion: 0.2
      dt: 1.0
      thread_num: 8
